src/pages/Items/BestProductList.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import BestProductItem from "./BestProductItem";
+import { getProductItem } from "./api";
+
+interface BestProductListProps {
+  pageSize: number;
+}
+
+interface Product {
+  createdAt: string;
+  description: string;
+  favoriteCount: number;
+  id: number;
+  images: string[];
+  name: string;
+  ownerId: number;
+  price: number;
+  tags: string[];
+  updatedAt: string;
+}
+
+const BestProductList = ({ pageSize }: BestProductListProps) => {
+  const [product, setProduct] = useState<Product[]>([
+    {
+      createdAt: "",
+      description: "",
+      favoriteCount: 0,
+      id: 0,
+      images: [],
+      name: "",
+      ownerId: 0,
+      price: 0,
+      tags: [],
+      updatedAt: "",
+    },
+  ]);
+
+  useEffect(() => {
+    const getBestProduct = async () => {
+      const data = await getProductItem(1, pageSize, "favorite");
+      setProduct(data.list);
```

Comment:
현재는 data에서 list 부분만 사용하고 있으므로 아래와 같이 구조분해 할당을 하용해서 표현하면 더 좋을 것 같아요!

```suggestion
      const { list } = await getProductItem(1, pageSize, "favorite");
      setProduct(list);
```